sycl/source/detail/device_info.hpp

```diff
@@ -233,6 +233,28 @@ template <> struct get_device_info<bool, info::device::queue_profiling> {
   }
 };
 
+// Specialization for atomic64 that is necessary because
+// PI_DEVICE_INFO_ATOMIC_64 isn't implemented for backend other than cuda.
+// TODO the if-statement can be removed when the other backends support
+// PI_DEVICE_INFO_ATOMIC_64.
```

Comment:
see comment below, just allow backend to "not know" the value and gracefully react to an error
we don't need to have backend specific paths in SYCL RT for this

Reply:
OK thanks. Done.